server.js

```diff
@@ -14,9 +14,17 @@ migrationsManager
       enabled: !!config.common.rollbar.accessToken,
       environment: config.common.rollbar.environment || config.environment
     }); */
-    new ApolloServer({ schema }).listen(port).then(({ url, subscriptionsUrl }) => {
-      logger.info(`🚀 Server ready at ${url}`);
-      logger.info(`🚀 Subscriptions ready at ${subscriptionsUrl}`);
+    new ApolloServer({
+      schema,
+      formatError: err => ({
+        message: err.message,
+        statusCode: err.extensions.code
+      })
```

Comment:
- Abstract `formatError` to a function, reference it here. 
- Make sure to log errors in the function.
- WRITE TEST! This is very important, you should write test for every new functionality.

Reply:
Also, make sure to catch errors in every service/resolver you are using. All errors should be catched and parsed before pass it to Apollo. For example:

- Return 404 when required resource is not available.
- Return 400 when input is invalid.
- Return 503 when database connection fails.
- And so on...